packages/core/content-type-builder/admin/src/components/FormModal/attributes/types.js

```diff
@@ -172,7 +173,13 @@ const types = {
           name: 'doesNotStartWithANumber',
           message: getTrad('error.validation.enum-number'),
           test: values => !values.some(startsWithANumber),
+        })
+        .test({
+          name: 'matchesRegex',
+          message: getTrad('error.validation.match-regex'),
+          test: values => values.every(matchRegex),
```

Comment:
The regex you need already exists, so there is no need to duplicate the knowledge about it:

```suggestion
          test: values => values.every(vENUM_REGEX.test),
```

What do you think?